src/plugins/loan-channel-query-map.js

```diff
@@ -0,0 +1,316 @@
+export default {
```

Comment:
New Map of existing loan channel query params converted to new Algolia facet query params where possible.

This file covers all of the urls that are shown on the lend menu, category grid, all categories page and lend by category page. Almost all of them can be recreated using new Algolia facet params.

Also note some urls aren't supported yet and should probably have fallbacks setup similar to the 'livestock' url which uses a facet we don't have yet.

Reply:
This is awesome!